apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -315,7 +352,10 @@ class VisualizerModal extends React.Component {
                     : msg.dataVisualizerValues()
                 }
                 options={this.props.tableColumns}
-                disabledOptions={disabledOptions}
+                disabledOptions={_.union(
+                  disabledOptions,
```

Comment:
What do you think of renaming `disabledOptions` to `disabledNonNumericOptions` to distinguish from disabled options that are too big?